### ISSUE TITLE
Hyperparameters used for PanNuke training/validation

### ISSUE THREAD

**Issue by kjolly02** [vqdang/hover_net#218]
Hello! Currently working on replicating performance using the repository on the PanNuke dataset. Was wondering if the base hyperparameters (LR/scheduler, epochs for each phase, etc) used in the repo and in the paper were kept constant for training and validation on PanNuke.

Thanks for the help.
